src/components/contact/assets/contact.scss

```diff
@@ -58,23 +58,11 @@ section.contact {
   color: inherit;
   text-decoration: none;
   border-bottom: 1px solid $black;
+  transition: background 300ms, color 300ms;
```

Comment:
What's the transition on these? Just linear?

Reply:
I would recommend ease-in or ease-out.

Reply:
oh sorry! must have missed while typing.  doing it now

Reply:
done! :)